source/uwp/Renderer/lib/AdaptiveTextBlockRenderer.h

```diff
@@ -3,24 +3,33 @@
 #include "AdaptiveCards.Uwp.h"
 #include "Enums.h"
 #include "TextBlock.h"
+#include "XamlBuilder.h"
 
-namespace AdaptiveCards { namespace Uwp
-{
-    class AdaptiveTextBlockRenderer :
-        public Microsoft::WRL::RuntimeClass<
-        Microsoft::WRL::RuntimeClassFlags<Microsoft::WRL::RuntimeClassType::WinRtClassicComMix>,
-        ABI::AdaptiveCards::Uwp::IAdaptiveElementRenderer>
+namespace AdaptiveCards {
+    namespace Uwp
```

Comment:
The other classes all mostly have this on one line, so probably makes sense to keep this one that way as well for consistency.